telemetry/definitions/commonDefinitions.json

```diff
@@ -1273,6 +1273,18 @@
                 }
             ],
             "passive": true
+        },
+        {
+            "name": "deeplink_open",
```

Comment:
Should this be passive by default?

Reply:
links are only opened when users click on them, so i think it's appropriate to keep this as an active metric

Reply:
sync'd offline; will make passive by default in case that use-case is more prevalent in the future